cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -251,6 +251,12 @@ set(
     "--user-config=${boost_user_jam}"
     --with-@HUNTER_PACKAGE_COMPONENT@
 )
+if("@MINGW@")
+  if("@CMAKE_CROSSCOMPILING@")
+     # cross compile from "linux" to "windows" using mingw
+     set(build_opts target-os=windows ${build_opts})
```

Comment:
What if we will use `target-os=windows` on Windows? If this will work we can simply remove `if(CMAKE_CROSSCOMPILING)`.